source/jormungandr/jormungandr/interfaces/v1/EquipmentReports.py

```diff
@@ -68,18 +70,30 @@ def __init__(self):
     def options(self, **kwargs):
         return self.api_description(**kwargs)
 
-    def get(self, region=None, lon=None, lat=None, uri=None):
+    def _create_filter_equipment(self, instance):
+        code_types = [
```

Comment:
Same minor here. You could use a generator instead of a list.

Reply:
@xlqian has a bigger idea to reshape this function.